mapclientplugins/segmentationstep/widgets/segmentationtabwidget.py

```diff
@@ -33,10 +33,12 @@ def __init__(self, parent=None):
 
         self.setAcceptDrops(True)
 
-        self._animation_increase = QtCore.QPropertyAnimation(self, 'maximumWidth')
+        self._animation_increase = QtCore.QPropertyAnimation(
+            self, QtCore.QByteArray("maximumWidth".encode()))
```

Comment:
We don't actually need to create a `QtCore.QByteArray` here we can just make the string a bytes object by doing `b"maximumWidth"` I think this is a better fix for this problem.

Reply:
The same goes for other places.

Reply:
Yes, that's much easier. Done.